nixnet/database/_frame.py

```diff
@@ -239,28 +243,24 @@ def mux_subframes(self):
         return self._mux_subframes
 
     @property
-    def pdu_refs(self):
-        return _props.get_frame_pdu_refs(self._handle)
-
-    @pdu_refs.setter
-    def pdu_refs(self, value):
-        _props.set_frame_pdu_refs(self._handle, value)
-
-    @property
-    def pdu_start_bits(self):
-        return _props.get_frame_pdu_start_bits(self._handle)
-
-    @pdu_start_bits.setter
-    def pdu_start_bits(self, value):
-        _props.set_frame_pdu_start_bits(self._handle, value)
-
-    @property
-    def pdu_update_bits(self):
-        return _props.get_frame_pdu_update_bits(self._handle)
-
-    @pdu_update_bits.setter
-    def pdu_update_bits(self, value):
-        _props.set_frame_pdu_update_bits(self._handle, value)
+    def pdus(self):
```

Comment:
Isn't Damon creating documentation for these?  Are you two's commits going to step on each others toes?

Reply:
Yep. I'll coordinate with damon on how to order these.

Reply:
@trstamper #237 has been pulled, so whenever you're ready, you'll want to pull down master and merge it with your changes. There will be conflicts to resolve.